app/components/badge.tsx

```
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export type BadgeProps = { href: string, imgSrc: string, alt: string }
+
+export function Badge({ href, imgSrc, alt }: BadgeProps) {
+    return (
+        <a href={href} rel="noreferrer noopener" target="_blank">
+            <img alt={alt} loading="lazy" decoding="async" data-nimg="1" style={{ color: 'transparent' }} src={imgSrc}>
+            </img>
```

Biome (1.9.4)

[error] img is a void element tag and must not have children.

Unsafe fix: Remove the children.


(lint/correctness/noVoidElementsWithChildren)